datasets/au/dfat_sanctions/au_dfat_sanctions.yml

```diff
@@ -89,6 +89,82 @@ lookups:
           - n/a
           - na
         value: null
+      - match: Special Karaj Road, Mehrabad Airport, Tehran, Iran; Sepahbod Gharani 36, Tehran, Iran; Plant No. 1, opp. Of 2nd Phase of Shahrak e Ekbatan, Karaj Special Road, Mehrabad Airport, 10000 Tehran, Iran; P.O. Box 83145/311, Tehran, Iran; P.O. Box 14155/1449, Tehran, Iran; Ekbatan City, Karaj Rd, Azadi Sq, Tehran, Iran
+        values: 
+          - Special Karaj Road, Mehrabad Airport, Tehran, Iran
+          - Sepahbod Gharani 36, Tehran, Iran
+          - Plant No. 1, opp. Of 2nd Phase of Shahrak e Ekbatan, Karaj Special Road, Mehrabad Airport, 10000 Tehran, Iran
+          - P.O. Box 83145/311, Tehran, Iran
+          - P.O. Box 14155/1449, Tehran, Iran
+          - Ekbatan City, Karaj Rd, Azadi Sq, Tehran, Iran
+      - match: PO Box 83145 311, 28 km Esfahan – Tehran Freeway, Shahin Shahr, Esfahan, Iran; PO Box 14155 5568, No 27 Ahahamat Ave, Vallie Asr Square, Tehran 15946, Iran; PO Box 81465 935, Esfahan, Iran; Shahih Shar Industrial Zone, Isfahan, Iran; PO Box 8140, No 107, Sepahbod Gharany Ave, Tehran, Iran
+        values:
+          - PO Box 83145 311, 28 km Esfahan – Tehran Freeway, Shahin Shahr, Esfahan, Iran
+          - PO Box 14155 5568, No 27 Ahahamat Ave, Vallie Asr Square, Tehran 15946, Iran
+          - PO Box 81465 935, Esfahan, Iran
+          - Shahih Shar Industrial Zone, Isfahan, Iran
+          - PO Box 8140, No 107, Sepahbod Gharany Ave, Tehran, Iran
+      - match: Unit 2, No. 3, 2nd Alley, Asad Abadi St, Vali-e asr St, 14316 Iran (Central Office); Flat 3, no. 3, 2nd Street, asad Abadi Ave, Tehran 14316 Iran (office); 3, 2nd Street, Assad-Abadi Ave., Vali-Asr Ave. 14316 Tehran, Iran; No. 3, 2nd St., Yousefabad Forked Rod., Vali-e Asr Ave., Tehran 14316 Iran; G.T.B. Complex, 44th Km. of Kashan-Delijan Road, Kashan (Factory); Km 44 Kashan-Delijan Rd., Kashan, Iran (office)
+        values:
+          - Unit 2, No. 3, 2nd Alley, Asad Abadi St, Vali-e asr St, 14316 Iran (Central Office)
+          - Flat 3, no. 3, 2nd Street, asad Abadi Ave, Tehran 14316 Iran (office)
+          - 3, 2nd Street, Assad-Abadi Ave., Vali-Asr Ave. 14316 Tehran, Iran
+          - No. 3, 2nd St., Yousefabad Forked Rod., Vali-e Asr Ave., Tehran 14316 Iran
+          - G.T.B. Complex, 44th Km. of Kashan-Delijan Road, Kashan (Factory)
+          - Km 44 Kashan-Delijan Rd., Kashan, Iran (office)
+      - match: Chohar Mir Road, Kandahar Bazaar, Quetta City, Baluchistan Province, Pakistan; ii) Room number 1, Abdul Sattar Plaza, Hafiz Saleem Street, Munsafi Road, Quetta, Baluchistan Province, Pakistan; iii) Shop number 3, Dr Bano Road, Quetta, Baluchistan Province, Pakistan; iv) Office number 3, Near Fatima Jinnah Road, Dr Bano Road, Quetta, Baluchistan Province, Pakistan;
```

Comment:
could you try a multisplit(..., ["ii)", "iii)", ..., ";"] on this? It might not fix everything, but it might improve more entities than just these

Reply:
we wanna improve matching with addresses in the near future so this is potentially useful matching data